### PR TITLE
v1.17 release

### PR DESCRIPTION
**Pull request by Sjors**
- [x] set base to `v1.16-release` once one PR is merged into this 
- [x] set base to master after `v1.16-release` is merged into master
- [x] merge after `v1.17` is deployed to production.

### CODE REVIEW

File: app/partials/navigation.pug [hunk 1]
```diff
@@ -28,23 +28,23 @@
             li
               a(href="{{rootURL}}", translate="HOME" target="_blank" rel="noopener noreferrer")
             li
-              a(href="{{rootURL}}charts", translate="CHARTS" target="_blank" rel="noopener noreferrer")
+              a(href="/charts", translate="CHARTS" target="_blank" rel="noopener noreferrer")
```
Comment (Sjors) [inline app/partials/navigation.pug]: Why are we no longer using `rootURL` here? (cc @plondon @jtormey)

File: app/partials/navigation.pug [hunk 2]
```diff
@@ -28,23 +28,23 @@
...
-              a(href="{{rootURL}}stats", translate="STATS" target="_blank" rel="noopener noreferrer")
+              a(href="/stats", translate="STATS" target="_blank" rel="noopener noreferrer")
             li
-              a(href="{{rootURL}}markets", translate="MARKETS" target="_blank" rel="noopener noreferrer")
+              a(href="/markets", translate="MARKETS" target="_blank" rel="noopener noreferrer")
             li
-              a(href="{{rootURL}}api", translate="API" target="_blank" rel="noopener noreferrer")
+              a(href="/api", translate="API" target="_blank" rel="noopener noreferrer")
         li.item.visible-xs
           a(href="{{rootURL}}", translate="HOME" target="_blank" rel="noopener noreferrer")
```
Comment (Sjors) [inline app/partials/navigation.pug]: And why _do_ we use it here?

File: app/partials/navigation.pug (hunk 1 above)
Comment (plondon) [inline app/partials/navigation.pug]: Because https://blockchain.info/wallet/charts is wrong.

File: app/partials/navigation.pug (hunk 2 above)
Comment (plondon) [inline app/partials/navigation.pug]: Because https://blockchain.info/wallet/ is right.

**Pull request merged**